.gitignore

```diff
@@ -15,6 +15,10 @@
 /db/*.sqlite3
 /db/*.sqlite3-journal
 
+# Ignore database.yml file
+
+/config/database.yml
```

Comment:
@mattwr18 I can help you to get the configuration you need in your computer for using this `database.yml` there is no need to remove it :+1: 

Alternatively you can take a look here https://gist.github.com/nisevi/49da4b76da393b9deaeafa0759334c6c :robot:

Reply:
I added it to the ` .gitignore` because my password is different. I have `postgres` working and am able to access the database. If there is a preferred way of doing this, I can change it. 
I know with RFM there is a `database.template.yml` so that people can copy it to a `database.yml` and use their credentials.

Reply:
I agree that you can have a different configuration on your computer but that is not enough to make a change that will impact on the configuration of all developers contributing to this project, also in a PR that it has nothing to do with the story on Pivotal Tracker.

To be more polite about the change that you are trying to make, you could create another story and discuss about it in the Slack channel. In this way all devs will have the opportunity to make their statements :+1: .

Reply:
Sure. I didn't make this change without first talking to @tansaku. I'm happy to change it though.